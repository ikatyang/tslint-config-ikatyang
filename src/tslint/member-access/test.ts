// good

class good1 {
  something: any;
}

// bad

class bad1 {
  public something: any;
}
