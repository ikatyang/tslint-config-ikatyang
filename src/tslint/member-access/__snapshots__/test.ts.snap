// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`lint 1`] = `
// good

class good1 {
  something: any;
}

// bad

class bad1 {
  public something: any;
  ~~~~~~                 ['public' is implicit.]
}

`;
